xml/hardening.xml

```diff
@@ -340,31 +340,30 @@
   <title>Common Criteria</title>
   <info>
    <abstract>
-    <para></para>
-   </abstract>
-  </info>
-  <sect1 xml:id="sec-common-criteria-introduction">
-   <title>Introduction</title>
-
-   <para>
+    <para>
     Common Criteria is the best known and most widely used methodology to
     evaluate and measure the security value of an IT product. The methodology
     aims to be independent, as an independent laboratory conducts the
     evaluation, which a certification body will certify afterward. Security
     Functional Requirements (SFR) are summarized in so-called Protection
-    Profiles (PP), which allows the comparison of security functions of
-    different products if the definition of the Security Target (ST) (which
-    typically uses a reference to the PP if one exists that fits the purpose
-    of the product) and the Evaluation Assurance Levels are comparable.
+    Profiles (PP). If the definition of a Security Target (ST) and the
+    Evaluation Assurance Levels (EAL) are comparable, this allows the comparison of
+    security functions of different products. (The definition of a Security Target
+    typically references the PP&mdash;if one exists that fits the purpose
+    of the product).
    </para>
+   </abstract>
+  </info>
+  <sect1 xml:id="sec-common-criteria-introduction">
+   <title>Introduction</title>
 
    <para>
     A clear definition of security in IT products is challenging. Security
     should be considered a process that never ends, not a static condition
-    that can be met or not. A Common Criteria certificate (below EA7) does not
-    make a clear statement about error proneness of the system (while many of
-    the flaws that exist specifically in operating systems are
-    security-relevant), but it adds an important value to the product that
+    that can be met or not. A Common Criteria certificate (below EA7)
+    <remark>taroth 2019-11-07: shouldn't that read *EAL7*?</remark>does not
```

Comment:
Yes, it is EAL7

Reply:
Yep, also confirmed by Ivan in Bugzilla, fixed.